docs/docs/introduction/getting-started.mdx

```diff
@@ -64,54 +97,64 @@ To start all the default Instant OpenHIE packages, select a deployment platform
 >
 <TabItem value="dockerCompose">
 
-**Before proceeding**, make sure you are in the main `/instant` directory. This directory contains the `instant.sh` script.
+If you have the Instant [OpenHIE codebase](https://github.com/openhie/instant) checked-out make sure you are in the main `/instant` directory.
 
 1. To start up the system, execute the following command:
 
     ```sh
-    ./instant.sh up docker
+    yarn
+    yarn docker:instant init
```

Comment:
I think lets add in the specific environment we are targeting just to be a bit more verbose and inform the reader its possible to set the target environment instead of just falling back to the default